docs/erasure/README.md

```diff
@@ -22,6 +22,14 @@ Bit Rot, also known as data rot or silent data corruption is a data loss issue f
 
 MinIO's erasure coded backend uses high speed [HighwayHash](https://github.com/minio/highwayhash) checksums to protect against Bit Rot.
 
+## How are drives used for Erasure Code?
+
+The number of drives you present to MinIO server must be a multiple of 4, 6, 8, 10, 12, 14 or 16.  The drives are divided into erasure coding sets, and each object is striped over a single set. Therefore, each object is spread across no more than 16 drives.
```

Comment:
```suggestion
The number of drives you present to MinIO server must be divisible by either 4, 6, 8, 10, 12, 14 or 16.  The drives are divided into erasure coding sets, and each object is striped over a single set. Therefore, each object is spread across no more than 16 drives.
```

Reply:
I will reword as:

"MinIO divides the drives you provide into erasure-coding sets of 4, 6, 8, 10, 12, 14 or 16 drives.  Therefore, the number of drives you present must be a multiple of one of these numbers; and any single object is spread across no more than 16 drives."

Reply:
Makes sense